src/generic/data/api.ts

```diff
@@ -24,9 +28,8 @@ export async function getOrganizations() {
 
 /**
  * Get's course rerun data.
- * @returns {Promise<Object>}
  */
-export async function getCourseRerun(courseId) {
+export async function getCourseRerun(courseId: string): Promise<unknown> {
```

Comment:
For now, we are using `unknown`. If/when we migrate the code that uses this, we could create a type for the API return.